racket/collects/racket/init.rkt

```diff
@@ -12,3 +12,7 @@
 (provide (all-from-out racket
                        racket/enter
                        racket/help))
+
+;; Set the default replt to XREPL
+(with-handlers ([exn:fail? void])
```

Comment:
Fair. Well also iirc, xrepl actually swallows up the error itself if libedit or libreadline are not installed, and works normally, just without readline support.

Either way though, this doesn't really matter, so I'll just change it to the other way (the second one you suggested in slack using `collection-file-path` and call it good.